xarray/tests/test_computation.py

```diff
@@ -695,8 +695,6 @@ def check(x, y):
     check(data_array, 0 * data_array)
     check(data_array, 0 * data_array[0])
     check(data_array[:, 0], 0 * data_array[0])
-    with raises_regex(ValueError, "with different chunksize present"):
```

Comment:
change this to `with pytest.warns(...)` instead?

Reply:
Yes, thanks, probably better than just removing.